projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/components/positioned-badge-docs/positioned-badge-docs.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component } from '@angular/core';
+import { SubNavService } from 'projects/go-style-guide/src/app/shared/components/sub-nav/sub-nav.service';
+
+@Component({
+  selector: 'app-positioned-badge-docs',
+  templateUrl: './positioned-badge-docs.component.html',
+})
+export class PositionedBadgeDocsComponent {
+  componentBindings: string = `
+  @Input() badgeData: string;
+  @Input() badgeColor: string = 'neutral';
+  @Input() displayData: boolean = true;
+  `;
+
+  iconBadge: string = `
+  <go-badge badgeData="12">
```

Comment:
Please indent everything within the backticks for all these string objects below - not trying to nit-pick just trying to keep the code legible!

Reply:
we need above and below one liner space to keep string aligned inside the <code></code>.

Reply:
oh okay, that makes sense